accepted/0000-sle-micro-reboot-action.md

```diff
@@ -0,0 +1,56 @@
+- Feature Name: SLE Micro - reboot action
+- Start Date: 2021-09-29
+
+# Summary
+[summary]: #summary
+
+Provide a suitable approach to handle reboot action in SLE Micro
+
+# Motivation
+Transactional systems, like SLE micro, use `transactional-update` to perform management actions including reboot. Transactional update itself supports several different reboot methods, configurable via the REBOOT_METHOD configuration option in `transactional-update.conf`. In order to put Suma in control of the reboot of its managed systems, it is necessary to configure transactional systems in a proper way and carry out the reboot action according to their particularities.
+
+# Detailed design
+[design]: #detailed-design
+
+This proposed solution is based on the following steps:
+ - Configure transactional systems to use `systemd` as reboot method
+ - Disable `transactional-update-timer` and `rebootmgr` services.
+ - Use `transactional-update` module to perform the reboot in transactional systems
+
+### Systemd as reboot method
+As mentioned, `transactional-update` supports different reboot methods. By default, `rebootmgrd` will be used to reboot the system according to the configured policies if the service is running, otherwise systemctl reboot will be called.
```

Comment:
```suggestion
As mentioned, `transactional-update` supports different reboot methods. By default, reboot method is set to `auto`, which means that `rebootmgrd` will be used to reboot the system according to the configured policies if the service is running, otherwise systemctl reboot will be called.
```